packages/compiler-sfc/__tests__/templateTransformAssetUrl.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { generate, parse, transform } from '@vue/compiler-core'
+import { transformAssetUrl } from '../src/templateTransformAssetUrl'
+import { transformElement } from '../../compiler-core/src/transforms/transformElement'
+import { transformBind } from '../../compiler-core/src/transforms/vBind'
+
+function compileWithAssetUrls(template: string) {
+  const ast = parse(template)
+  transform(ast, {
+    nodeTransforms: [transformAssetUrl, transformElement],
+    directiveTransforms: {
+      bind: transformBind
+    }
+  })
+  return generate(ast)
```

Comment:
Should generate with `mode: 'module'` here since this feature is only used in module mode.